miniapp/miniapp_eigensolver.cpp

```diff
@@ -176,6 +223,10 @@ int main(int argc, char** argv) {
   desc_commandline.add_options()
     ("matrix-size",  value<SizeType>()   ->default_value(4096), "Matrix size")
     ("block-size",   value<SizeType>()   ->default_value( 256), "Block cyclic distribution size")
+#ifdef DLAF_WITH_HDF5
+    ("input-file",   value<std::string>()                     , "Load matrix from given HDF5 file")
+    ("output-file",  value<std::string>()                     , "Save eigenvectors and eigenvalues to given HDF5 file")
```

Comment:
I don't remember why I didn't use `std::filesystem::path` in the beginning (I mean in `miniapp_tridiag_solver`), probably just forgot about it. 😅